bot/db/latest_mention.py

```diff
@@ -3,8 +3,9 @@
 from .neo4j_connection import Neo4jConnection
 
 
-def get_latest_mention_in_past_7_days(
+def get_latest_mention_since(
     user_id: str,
+    since: int = get_x_days_ago_UTC_timestamp(7)
```

Comment:
I'd suggest to change the function name to `get_days_ago_UTC_timestamp` and put the explanation in the documentation of the function.

Reply:
I believe the one we already have makes more sense since everyone understand it without needing to read the documentation